src/applications/vaos/services/appointment/transformers.js

```diff
@@ -415,64 +415,85 @@ function setContained(appt) {
       return null;
     }
     case APPOINTMENT_TYPES.request: {
+      const contained = [createPatientResourceFromRequest(appt)];
+
       if (appt.visitType === 'Video Conference') {
-        return [
-          {
-            resourceType: 'HealthcareService',
-            characteristic: [
-              {
-                coding: getVideoType(appt),
-              },
-            ],
-          },
-        ];
+        contained.push({
+          resourceType: 'HealthcareService',
+          characteristic: [
+            {
+              coding: getVideoType(appt),
+            },
+          ],
+        });
       }
-      return null;
+
+      return contained;
     }
     case APPOINTMENT_TYPES.ccRequest: {
-      return appt.ccAppointmentRequest.preferredProviders.map(provider => {
-        const participant = {
-          actor: {
-            name: provider.practiceName,
-            // TODO: Map to participant.actor.Practitioner field.
-            firstName: provider.firstName,
-            lastName: provider.lastName,
-          },
-        };
-
-        if (provider.address) {
-          const address = {
-            line: [provider.address?.street],
-            city: provider.address?.city,
-            state: provider.address?.state,
-            postalCode: provider.address?.zipCode,
-          };
-          participant.actor.address = address;
-        }
+      const contained = [createPatientResourceFromRequest(appt)];
+      appt.ccAppointmentRequest.preferredProviders.forEach(
+        (provider, index) => {
+          const address = [];
+          if (provider.address) {
+            address.push({
+              line: [provider.address?.street],
+              city: provider.address?.city,
+              state: provider.address?.state,
+              postalCode: provider.address?.zipCode,
+            });
+          }
+
+          contained.push({
+            resourceType: 'Practitioner',
+            id: `cc-practitioner-${appt.id}-${index}`,
```

Comment:
Where does this naming schema come from, or is it just for our reference?

Reply:
Just for our reference. I wanted to make a unique id so that we could use that as `key` props when necessary, since that's what'd we normally try to use for `key`s.